frontend/app/services/TouchpointBackend.scala

```diff
@@ -126,6 +119,18 @@ object TouchpointBackend extends LazyLogging {
     )
   }
 
+  case class Resolution(
+    backend: TouchpointBackend,
+    typ: BackendType,
+    validTestUserCredentialOpt: Option[TestUserCredentialType[_]]
+  )
+}
+
+class TouchpointBackendProvider(implicit val system: ActorSystem, implicit val executionContext: ExecutionContext, implicit val wsClient: WSClient) extends LazyLogging {
```

Comment:
I called this TouchpointBackends in subs frontned

Reply:
I will rename for consistency